arrow-site/doc/docs/arrow-core/datatypes/intro/README.md

```diff
@@ -10,35 +10,35 @@ permalink: /docs/datatypes/intro/
 beginner
 
 A datatype is a an abstraction that encapsulates one reusable coding pattern.
-These solutions have a canonical implementation that is generalised for all possible uses.
+These solutions have a canonical implementation that is generalized for all possible uses.
```

Comment:
Generalised / generalized is British vs. US. I don't suppose we have a standard when it comes to documentation. (As a Canadian, my English would be all over both.)